client/my-sites/plans-comparison/plans-comparison-features.tsx

```diff
@@ -259,7 +259,9 @@ export const planComparisonFeatures: PlanComparisonFeature[] = [
 			return translate( 'Visits per month' );
 		},
 		get description() {
-			return translate( 'Max visits per month.' );
+			return translate(
+				'WordPress Pro includes a generous 100,000 visits per month, which is 4x the limit for similarly priced plans at other popular managed WordPress hosts. Additional traffic can be purchased in 100,000 increments.'
```

Comment:
Should we be using `WordPress.com Pro` instead for referring to the new plan? I believe this was the consensus in the end in a relevant P2 discussion (pdgrnI-rs-p2#comment-752)